src/main/resources/swagger/api-docs.yaml

```diff
@@ -4115,10 +4115,30 @@ definitions:
       ok:
         type: boolean
         description: whether any system(s) need attention
-
+        default: false
```

Comment:
I don't think you need `default` here at all - it doesn't add any value to the swagger UI.